backend/ttrs/views.py

```diff
@@ -150,3 +150,13 @@ class MajorDetail(generics.RetrieveAPIView):
     queryset = Major.objects.all()
     serializer_class = MajorSerializer
     permission_classes = (AllowAny,)
+
+from .recommend import recommend
+
+class RecommendView(generics.RetrieveAPIView):
+    permission_classes = (AllowAny,)
+
+    def get(self, request):
+        options = request.query_params.get('options')
+        return Response(recommend(options))
```

Comment:
I suggest that send student information to the recommend at this time.
The student information can be acquired by requester's authentication.
But, just leave this, I will implement this part because I implemented `IsStudent` permission and know how to get student information from the requester's information.